core/src/epicli/data/common/defaults/configuration/kafka.yml

```diff
@@ -4,10 +4,10 @@ name: default
 specification:
 
   kafka_var:
-    version: 2.3.1
+    version: 2.6.0
     scala:
       version: 2.12
-    file_name: "kafka_2.12-2.3.1.tgz"
+    file_name: "kafka_2.12-2.6.0.tgz"
```

Comment:
If we don't allow user to change/choose the component versions, It should be moved to the role defaults. The same for any other parameters that are not allowed/supposed to be changed by user.

Reply:
Moved to defaults :)